id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/security/IdRepoSecurityManager.java

```diff
@@ -284,8 +285,8 @@ public static String getUser() {
 			}
 		} catch (RestServiceException e) {
 			mosipLogger.error(IdRepoSecurityManager.getUser(), ID_REPO_SECURITY_MANAGER, ENCRYPT_DECRYPT_DATA,
-					e.getErrorText());
-			throw new IdRepoAppException(ENCRYPTION_DECRYPTION_FAILED, e);
+					ExceptionUtils.getStackTrace(e));
```

Comment:
Do you need to stacktrace? just sending 'e' is not good enough?

Reply:
@gsasikumar stacktrace is for only logging purpose. It is not sent as root cause here.